{"uuid": "c3ff5554-532e-4085-8072-1e0cc35fe6c4", "children": ["9735d7b7-29b3-49aa-ad4c-a3449191ec4d"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\test_审批流程.py\", line 63, in login\n    raise e\n  File \"C:\\Users\\Administrator\\Desktop\\test\\test_审批流程.py\", line 39, in login\n    captcha_data = captcha_img.get_attribute(\"src\").split(\",\")[1]\n                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^\n"}, "attachments": [{"name": "登录失败截图", "source": "0750df8d-8068-40f9-b973-e92dce652383-attachment.png", "type": "image/png"}], "start": 1751618991185, "stop": 1751618993598}], "start": 1751618991184, "stop": 1751618995008}